ndk/src/events/nip13/nip13.ts

```diff
@@ -0,0 +1,88 @@
+import { serializeEvent } from "nostr-tools";
+import type { UnsignedEvent } from "nostr-tools";
+import type { NDKEvent } from "../index.js";
+import Worker from './pow.worker.js';
```

Comment:
Did you mean to include this file with the PR?

Reply:
Uh, looks like line 9 of `/.gitignore` ignores `**/*.js'. So it never got committed.

I could try converting it to a TypeScript file but I've never gotten a TypeScript webworker to compile and work properly. I'm sure there must be a way to do it correctly... I'll look for a tutorial.